riemann/tx/tx.py

```diff
@@ -195,7 +195,14 @@ def from_bytes(VarInt, byte_string):
             raise ValueError('Malformed VarInt. Got: {}'
                              .format(byte_string.hex()))
 
-        return VarInt(utils.le2i(num))
```

Comment:
why create variable ret instead of just moving the ' return VarInt(utils.le2i(num))' line to after the new ValueError lines you added? i know it results in the same thing but could look cleaner without assigning the ret var